packages/calcite-components/src/demos/panel.html

```diff
@@ -282,6 +282,48 @@
         </div>
       </div>
 
+      <!-- closable with event prevented -->
```

Comment:
Is this necessary if we already have an automated test for it? I’m noticing our demo pages are growing without a clear structure and often duplicating coverage from E2E or screenshot stories.

Reply:
I think its nice to have this one in the demos but If you feel strongly I can remove it